src/pages/orders/orders-page/orders-page.js

```diff
@@ -1,11 +1,24 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+
 import { useStyles } from './orders-page.styles';
+import { getOrderList } from '../../../redux/orders/orders.actions';
 
 const OrdersPage = () => {
-  const classes = useStyles();
+  const styles = useStyles();
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(
+      getOrderList({
+        limit: 10,
+        skip: 2
```

Comment:
Do not use 'magic numbers', select this data from the store it contains in the `pagination` property

Reply:
fixed